src/Core/Geographic/Coordinates.js

```diff
@@ -224,8 +227,28 @@ export function convertValueToUnit(unitIn, unitOut, value) {
 function Coordinates(crs, ...coordinates) {
     this._values = new Float64Array(3);
     this.set(crs, ...coordinates);
+
+    Object.defineProperty(this, 'surfaceNormal',
```

Comment:
Let's steal the wording from your other PR: `geodeticSurfaceNormal` is a good name.

Reply:
Are you sure `geodetic` is a real word? Or do you mean `geodesic...`?